.travis.yml

```diff
@@ -13,21 +13,23 @@ before_script:
+        2k8Bwh77VofZiXb0zgPxyUojSaKEiH8+qObYWQ6y4vtBrfuctVIgR2HT9nuLkJ/3duko4vyYXy78zAhnGMdvrXKKMOOebxH2RO5tx0ccC2OLjnYrX/qMxY63v7tXnEa+e97Uax5/x3ZHUq48gE1xhSi9U9BTd0scbjv8xMqm5yIkMbdBhm60a/z/lyorCOc4hQEbY4mqSM5dgq0xWMz1Jd5fU16iXlVsI+vTaGrV1HsYVl3Ysn4Q40rX6/fAhTIhGvKLwXHwCak23ToQgW1pOihIcq/2N5XnNlflEkKlcQOmiGK9zxyXZOM+sy3YwA5EjUzm6BbkPbOqzNhtuUkL5m/T3NkdqEllhgwmFhDmLuN3yaq2J5n8kh60Z/QJJu2nlFHbZ+njxkOm2XYF2OyYn4nearyAyJLsOAoQEtcU+ftmWFukQeEXkvu6NP7OYLT6DOn/yB0YDIpMz1g6EFkuvBVNJ17JSQjAavpYRAeCfb+orep/hWv0jqfB9PFul2AOAInKvWl3GgVw8azJlLAFVOp3XRRod8aabAuXAslXnxoMANm/kPWah0NN02ptIGVOumas0IutThCMnd94OcymVMlMGqpce+bfIR7JlR2xcJElvBDNYZz5ZyFmNvzLuqmxMSd6pV2Tmzq+YsMR7NXkrb0MGujmLTilqhYYqffU9Bo=
     - secure: >-
-        Q3xN3fRvTQuy/HKlkYFFnVrFo01r6Q8zgzHHK8yKNKio9T/BM0+iIMYP8mRY+qvCQxCboXuUawG6gsxxT2Zn/p7SgNZ+UHq7DcLmocqxmECdGfqra6Hy9Y7BZVYPWlWIADkNUjI0RfOz69/3o/TAFlt4Cnw3BX3ip0o0Rri5/jzj0Nn+xSF+onyMnpH2gQvUE77MHvCuWTPki3R86Bz8RRzbTwKYwVa0oVca7jzdPtOjqzsnz8k9X0HVwQEpHGjqgTP3lg8EotON0rac/ayWs3J3bk9ye5AfvdTCYcZnDcz8POAN6FeC2Xey7oqQO4N8vFagru88mC3AmN67ZYqwI0fmDEYre20lnXFAFBFzsiu4FvKMgFYi/C5tG5ngSR5XcXwLskax82Un7yYYRzVbyhFSxxWZCqIC4cJ5d+E0Wex4eEYDwAlZ9AFP/hRJ2qrBVUbzXeGVMkHlg4qtdUIzHRkMUmjvEWVzTvk0Xa2wD+S0mYPcBwQcscoZy9zqdJM4hfbG9IGi02/DyrTOL3IuNH8WHHvGWS4ajDhSRkGCV4I5VtGKQ/Y3RA/pU6AaI7tVcDhbxOyIPKuOXO6PWuZOiJhhPQ/kceMkUXeh0TmsDtSBpe5OKCnjsdSn02lTT5yYf4nlVIVczP9p6zBerXorzZk1/S3CRBtqYZPgRrefbCw=
-node_js: stable
+        sTnjoXzKT5FKknJOyc0AqtvHf9dF31Pi+SEYdbanUpjQKK8wUqe3hVwf5Rf4qLFZsenR65o6M5Ual3rjyEB7WMVaU6x7N5ZEEZrXLQqVGq/uo8rtNi6CrmVzOPjgeKzcHOiAc2dPBp1ZVWGLjmjIBidItkXWBMBEOnmHeYofoX9Yi3p7UtT5o21rLphBSH7MXnBjW73jw0zW8TDNxjgc/PKypshsef0NfKScsbbRY5n0v2YTMua6rNXD9TDjBqw+OOnvjMdtRW7tY5ztVCnSwnTiHFDD1X6zkYShTrh1Aa3WZkPO9lM8NHek8WZAUkIb4c/UIHkE0dR80OgfVudx+tv5ACHwCaz7dV0kSy/mdKX3DWGtdIgAZmSVVYo2cXkgkpi36BlHltu7+xWB3dGYuWWIVw+mCstnouWv1xbPJd6DUfEV8+9WsQabaUU/Hs3LqAvrd4BRBDhanpDKxOmsDDhbRfovVDTdplWHUXx0JXy1xlJpCfc7hh0U+5HC1+G/3N5vTbAceFBf7Dz+sXrS48XnQ0iwlYBXgRtWXK2ysjKIRsSKenAR4rqyhqlnzGLIXDunsxcM82PNQLhltxj2PJ3k5gEcmsu+iB0UfsEYBVx3hbknWKLoLzjScRg9UdgK3eYm89nq0X00P4kjwemxkS1XhIeH8W4VaVmJzUkl+c0=
+node_js: '9'
 addons:
   firefox: latest
-  apt:
-    sources:
-      - google-chrome
-    packages:
-      - google-chrome-stable
-  sauce_connect: true
+  chrome: stable
 script:
   - xvfb-run polymer test
-  - >-
+  - |-
+    >-
     if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then polymer test -s 'default';
     fi
 dist: trusty
+install:
+  - npm install -g polymer-cli
+  - polymer install --variants
```

Comment:
The steps in this `install` section are at the beginning of the `before_script` section too; I'm betting one should be removed?

Reply:
You are correct. If I remove the install everything works. I think if you define what is in install it doesn't install anything else. e.g. the TS generator. I'm working on another run now